src/olympia/files/utils.py

```diff
@@ -73,8 +73,8 @@ def get_filepath(fileorpath):
         return fileorpath
     elif isinstance(fileorpath, DjangoFile):
         return fileorpath
-    elif hasattr(fileorpath, 'file_path'):  # File
-        return fileorpath.file_path
+    elif hasattr(fileorpath, 'file'):  # File
+        return fileorpath.file
```

Comment:
I know this isn't new code, but this is duck typing that's really class checking, so maybe it should really be `isinstance(fileorpath, File)`?

Reply:
Yeah... I really want to simplify/get rid of `get_file()` and `get_filepath()` but that breaks everything... I'll change it to do `isinstance()` checks and see what happens

Reply:
cee38ed57e0e7f4d095fd2adda3a1dcbb432eb93